src/apps/my-pipeline/client/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { state2props } from './state'
+import AddToPipelineForm from './AddToPipelineForm'
+import EditPipelineForm from './EditPipelineForm'
+
+function PipeLineFormView(props) {
+  const { pipelineItemId } = props
```

Comment:
OPTIONAL: can this be de-structured inline? with something like this? 
```suggestion
function PipeLineFormView({ pipelineItemId }, props) {
```

Reply:
Wouldn't that make props undefined?